west.yml

```diff
@@ -57,7 +57,7 @@ manifest:
       groups:
         - hal
     - name: hal_atmel
-      revision: 9f78f520f6cbb997e5b44fe8ab17dd5bf2448095
+      revision: e5d2a27c9c1146916e6853c74645663c23be17bc
```

Comment:
Please use `pull/17/head`.

Reply:
... will that magically resolve to a commit ID once the PR is merged? (or will we require a second PR here once that one is in place?)